clients/lightrider/lightrider-entry.js

```diff
@@ -38,6 +38,7 @@ async function runLighthouseInLR(connection, url, flags, lrOpts) {
   flags.disableStorageReset = true;
   flags.logLevel = flags.logLevel || 'info';
   flags.channel = 'lr';
+  // flags.output assumed to be unset by Lightrider
```

Comment:
yeah, probably just drop this line. At worst it's just extra work, and is just as assumed as every other property on `flags` could be (and we don't make a comment for any of those)

Reply:
should it instead throw if output is set to anything other than `undefined` or `json`? seems like it could be frustrating to figure that out on your own

 but you both seem to be on the same page that we "*definitely* don't need a check in the code" so I feel like I'm missing something

Reply:
> should it instead throw if output is set to anything other than undefined or json?

yeah, that's why I was suggesting removing the comment, because from this comment it *definitely* sounds like it should be an assertion, not a comment :)

AIUI, the `runnerResult.report` is ignored completely on the LR side now, it uses the LHR directly (with its own HTML reportGenerator bundle), so at worst a defined `output` is just extra work.

...but there are several possible things on `flags` that could add extra work and we're just assuming those are correct too, so I really don't think it's worth thinking about. Really it's something for the caller to worry about, either way anyways